src/simulation/m_monopole.fpp

```diff
@@ -370,44 +389,52 @@ contains
         real(kind(0d0)) :: hxnew_cyl, hynew_cyl
         real(kind(0d0)) :: sig
         real(kind(0d0)) :: f_delta
+        real(kind(0d0)) :: angle, ratio_x_r, ratio_y_r, ratio_z_r
```

Comment:
how about `ratio_r(1:3)` insteady of `x/y/z`

Reply:
I kind of prefer ratio_x_r because it's very explicit and clear that it's a ratio between x and r. But I can change it if you think `ratio_r(1:3)` suits the code better

Reply:
oh, i didn't even realize that's what it was supported to be. perhaps a better name is still needed then.

Reply:
Yes I changed it to xyz_to_r_ratios(1:3)